### ISSUE TITLE
membership-directory not working

### ISSUE THREAD

**Issue by Florencia007** [castlamp/zenbership#97]
Hi. I have installed successfully the membership directory but when I insert the widget  {-member_directory-} on a page it doesn't show anything. Anyone managed to make it work?

**Comment by samchadwick** [comment]
Make sure you are using v115 or higher.

**Comment by Websaint** [comment]
I am on v115 myself having updated the PPSD_options table in dataabase yet the member-directory plugin is still not working. Any ideas please!

**Comment by jbelelieu** [comment]
You can't include a plugin on a page using {-xxxxxxx-}; that is for widgets.

Plugins have routes associated with them. In this case, it is:

/directory
/directory/USERNAME_HERE

**Comment by slove05** [comment]
Yes sorry I had an incomplete file transfer for the plugin. This is why it was not rendering for me.

**Comment by Websaint** [comment]
@jbelelieu how do we now use this , am stalled! and I really want to use this,

**Comment by Websaint** [comment]
I have uninstalled and installed but the URL is nonexistent!
